### PR TITLE
fix: Add body validation

### PR DESCRIPTION
**Pull request by samuveth**
Requires: https://github.com/snapshot-labs/snapshot.js/pull/833

### Changes 
*_(Briefly describe the changes made in this PR)_*

1. Add better validation messages
2. Refactor create proposal validation 
3. Add error message for body limit
image

### How to test
*_(Explain how the changes can be tested, including any required setup steps)_*

1. Go to create proposal page

### To-Do
*_(List any outstanding tasks be addressed before or after this PR is merged)_*

- [x] Merge https://github.com/snapshot-labs/snapshot.js/pull/833

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### Notes
Making it show on button click would require much more work and result in a much larger PR. So I decided to not make it perfect for now

### CODE REVIEW

**Comment by ChaituVR** [comment]
Update snapshot.js version

**Comment by bonustrack** [comment]
Looks good!

**Comment by wa0x6e** [comment]
Should we trim() the text ? I can put 100 blank spaces and it passes the validation

File: src/composables/useFormSpaceProposal.ts [hunk 1]
```diff
@@ -70,15 +71,14 @@ export function useFormSpaceProposal() {
     userSelectedDateStart.value = false;
   }
 
-  const { getValidationMessage } = useFormValidation(schemas.proposal, form);
+  const validationErrors = computed(() => {
+    const errors = validateForm(schemas.proposal, form.value);
+    return errors;
```
Comment (wa0x6e) [inline src/composables/useFormSpaceProposal.ts:76]: Why do you assign to a const first, instead of directly returning the result ?

Comment (wa0x6e) [reply src/composables/useFormSpaceProposal.ts:76]: Apart from that, looks good

**Pull request merged**